config/routes.rb

```diff
@@ -3,4 +3,13 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root 'users#index'
+
+  get '/users', to: 'users#index'
+
+  get '/users/:id', to: 'users#show'
+
+  get '/users/:user_id/posts', to: 'posts#index'
+
+  get '/users/:user_id/posts/:post_id', to: 'posts#show'
```

Comment:
- [ **OPTIONAL**] You did an excellent job with the routes 👏. However, in order to make it easier to add nested routes and reduce typing, I recommend that you update your routes as follows. :point_down: 

```
  resources :users, only: [:index, :show] do
      resources :posts, only: [:index, :show]
  end
```